ShowPot/ShowPot/Presentation/Scene/Onboarding/OnboardingViewHolder.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  OnboardingViewHolder.swift
+//  ShowPot
+//
+//  Created by Daegeon Choi on 7/4/24.
+//
+
+import UIKit
+
+final class OnboardingViewHolder {
+    
+    let carouselLayout = UICollectionViewFlowLayout().then { carouselLayout in
+        carouselLayout.scrollDirection = .horizontal
+        carouselLayout.sectionInset = .init(top: 0, left: 0, bottom: 0, right: 0)
+        carouselLayout.minimumLineSpacing = 0
+        carouselLayout.itemSize = .init(width: UIScreen.main.bounds.width, height: 470)
+    }
```

Comment:
```suggestion
    private let carouselLayout = UICollectionViewFlowLayout().then { carouselLayout in
        carouselLayout.scrollDirection = .horizontal
        carouselLayout.sectionInset = .init(top: 0, left: 0, bottom: 0, right: 0)
        carouselLayout.minimumLineSpacing = 0
        carouselLayout.itemSize = .init(width: UIScreen.main.bounds.width, height: 470)
    }
```
carouselLayout은 OnboardingViewHolder에서만 쓰일꺼같아서 private을 추가해도 좋아보여요 !!